module/applications/combat/combat-tracker.mjs

```diff
@@ -23,4 +31,81 @@ export default class CombatTracker5e extends CombatTracker {
     if ( (btn.dataset.control === "rollInitiative") && combatant?.actor ) return combatant.actor.rollInitiativeDialog();
     return super._onCombatantControl(event);
   }
+
+  /* -------------------------------------------- */
+
+  /** @inheritdoc */
+  _contextMenu(html) {
+    if ( !(html instanceof HTMLElement) ) html = html[0];
+    new ContextMenu5e(
+      html.querySelector(".directory-list"), ".directory-item:not(.combatant-group)", this._getEntryContextOptions()
+    );
+  }
+
+  /* -------------------------------------------- */
+
+  /**
+   * Adjust initiative tracker to group combatants.
+   * @param {HTMLElement} html  The combat tracker being rendered.
+   */
+  renderGroups(html) {
+    if ( !this.viewed ) return;
+    const groups = this.viewed.createGroups();
+    const V13 = game.release.generation >= 13;
+    const list = html.querySelector(".directory-list, .combat-tracker");
+    for ( const [key, { combatants, expanded }] of groups.entries() ) {
+      const children = list.querySelectorAll(Array.from(combatants).map(c => `[data-combatant-id="${c.id}"]`).join(", "));
+      if ( !children.length ) continue;
+      const groupContainer = document.createElement("li");
+      groupContainer.classList.add("combatant", "combatant-group", "collapsible", "dnd5e2-collapsible");
+      if ( !V13 ) groupContainer.classlist.add("directory-item");
+      if ( !expanded ) groupContainer.classList.add("collapsed");
+
+      // Determine the count
+      let activeEntry;
+      for ( const [index, element] of children.entries() ) {
+        if ( element.classList.contains("active") ) activeEntry = index;
+      }
+      let count = game.i18n.format(`DND5E.COMBATANT.Counted.${getPluralRules().select(children.length)}`, {
+        number: formatNumber(children.length)
+      });
+      if ( activeEntry !== undefined ) {
```

Comment:
```suggestion
      if ( activeEntry ) {
```

Reply:
Active entry is the index, so `0` is a valid value.